### PR TITLE
fix forum link

### PR DESCRIPTION
**Pull request by xiaotinghe**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-1038/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-1038/

File: chapter_attention-mechanisms/attention-scoring-functions.md [hunk 1]
```diff
@@ -470,9 +470,9 @@ d2l.show_heatmaps(d2l.reshape(attention.attention_weights, (1, 1, 2, 10)),
...
 
 :begin_tab:`mxnet`
-[Discussions](https://discuss.d2l.ai/t/346)
+[Discussions](https://discuss.d2l.ai/t/5751)
 :end_tab:
 
 :begin_tab:`pytorch`
-[Discussions](https://discuss.d2l.ai/t/1064)
+[Discussions](https://discuss.d2l.ai/t/5752)
 :end_tab:
```
Comment (goldmermaid) [inline chapter_attention-mechanisms/attention-scoring-functions.md:478]: Seems that we are missing links for tensorflow?

**Comment by goldmermaid** [comment]
@xiaotinghe it seems that several tensorflow forum links are missing (e.g. in `chapter_attention-mechanisms`, etc.). Or am I missing anything?

**Comment by goldmermaid** [comment]
And the section link of each forum link here are using `http:` rather than `https:`, do we have to use the latter? (@xiaotinghe @astonzhang, please correct me if i was wrong)

**Comment by astonzhang** [comment]
> And the section link of each forum link here are using `http:` rather than `https:`, do we have to use the latter? (@xiaotinghe @astonzhang, please correct me if i was wrong)

http is fine in forums

**Comment by astonzhang** [comment]
> @xiaotinghe it seems that several tensorflow forum links are missing (e.g. in `chapter_attention-mechanisms`, etc.). Or am I missing anything?

@xiaotinghe I'm merging this PR for beta. Please add those links that @goldmermaid  mentioned in another PR. Thanks

**Pull request merged**